src/main/java/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,13 @@
+package main.java.duke;
+
+public class DukeException extends Exception{
```

Comment:
Perhaps, would it be better to create separate class for each error ("Oops" statement) that inherits from this DukeException class? Such that it displays the error message instead in the handleUserInput() method. - Just a suggestion! But it still looks good 👍

Reply:
Sounds great, thank you for the suggestion!